_features/fresh.html

```diff
@@ -2,6 +2,14 @@
 title: Fresh
 slug: fresh
 description: Always up-to-date.
+usecases:
+- As a web application developer, I want to deliver updates in a seamless way,
+  after the user visit the application next time.
+- As a user, I don't want to wait for the new version to be installed, I always
+  want the latest version available.
 technologies:
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>When your users visit your web, they are not prompted with a request to update,
+they simply see you brand new awesome UI with all the new functionality after
+the next visit. Progressive Web Applications behave exactly the same by default
+but they offer much more flexibility so if you prefer to prompt, you can do it.</p>
```

Comment:
Nits:
* web -> web app
* you brand new -> your brand new

I'd also strike "after the next visit," as this is not essential.

Also, it isn't clear what "exactly the same by default" means. Does it mean that PWAs degrade gracefully, working like classic web apps in older browsers?

And "offer much more flexibility" is similarly vague. Do you mean that the app lifecycle is customizable by the developer?